### ISSUE TITLE
No confirmation about when the file is uploaded completely!

### ISSUE THREAD

**Issue by lethalgit** [balderdashy/skipper-disk#9]
Is there a way by which we'll be able to see when the file upload gets completed to the respective folder?
The call back function(here whenDone function) is executed the moment the file started to upload, in that case we are loosing the track for when the file will be uploaded completely.

**Comment by sailsbot** [comment]
Thanks for posting, @lethalgit.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
